### ISSUE TITLE
bug when scans can't be loaded

### ISSUE THREAD

**Issue by berger-n** [buda-base/public-digital-library#945]
been able to reproduce this random bug from test when the scans can't be loaded (endless white spinner) 
seems to come from a 500 error when fetching the manifest
needs to be handled better in the client (at least an error message, an automatic retry maybe?)

![image](https://github.com/user-attachments/assets/f250be6f-e00c-420e-bb3b-c98b83a1cd44)

**Comment by eroux** [comment]
yes, automatic retry would be nice, although if it's integrated into Mirador it might not be so easy... but otherwise a proper error message with a way for users to retry would be nice!

**Comment by berger-n** [comment]
there might be an additional/alternative cause here:
(this is a long known issue from a [deprecated package](https://github.com/aeschylus/Isfahan) in use by the viewer
but it could be detected and trigger a reload which would successfully open the viewer I guess)

![image](https://github.com/user-attachments/assets/a4b52690-cf2f-4030-ba94-4690edd796a5)
